src/kixi/datastore/schemastore/validator.clj

```diff
@@ -54,6 +54,10 @@
   [definition _]
   (conformers/regex? (get-in definition [::ss/schema ::ss/pattern])))
 
+(defmethod resolve-schema "string"
+  [_ _]
+  conformers/-string?)
```

Comment:
then we get the name clash with core. It's not a private function. It just has the '-' prefix, it's not defn-'d